main.go

```diff
@@ -39,27 +40,33 @@ func init() {
 	rcommand.PersistentFlags().Int32VarP(&deployVar.Port, "port", "p", 8000, "port at which app should run")
 
 	rcommand.MarkFlagRequired("server")
+	countCommand.AddCommand(podCommand)
+
 }
 
 func main() {
-
 	rcommand.Execute()
 }
 
-var listCommand = &cobra.Command{
-	Use: "list",
+var countCommand = &cobra.Command{
+	Use:   "count",
```

Comment:
This change isn't relevant to this PR, please remove this.
We should be renaming the command in a different branch/PR.